mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/MessageMapperTest.java

```diff
@@ -365,7 +365,7 @@ public void findFirstUnseenMessageUidShouldReturnUid2WhenUid2isSeen() throws Mai
         messageMapper.updateFlags(benwaInboxMailbox, new FlagsUpdateCalculator(new Flags(Flags.Flag.SEEN), FlagsUpdateMode.REPLACE), MessageRange.one(message5.getUid()));
```

Comment:
We miss tests here for your brand new message manager method

Reply:
I know, will work on the tests today